### ISSUE TITLE
Game crashes

### ISSUE THREAD

**Issue by Booeraque** [OpenTTD/OpenTTD#9005]
I love your game and really hope you will develop it much better! <3

```
*** OpenTTD Crash Report ***

Crash at: Sat Apr 10 22:21:48 2021
In game date: 2095-05-19 (20)

Crash reason:
 Exception: C0000005
 Location:  00007FF6C6813215
 Message:   <none>

OpenTTD version:
 Version:    1.11.0 (0)
 NewGRF ver: 1b086d64
 Bits:       64
 Endian:     little
 Dedicated:  no
 Build date: Apr  1 2021 12:39:23

Registers:
 RAX: FEFFFFFFFF9F125F RBX: 00000195E73DCED0 RCX: 0000000000000000 RDX: 0000000000000000
 RSI: 00000195F2A3CFA0 RDI: 0000019583B72850 RBP: 0000000000000000 RSP: 000000821DAFF5E0
 R8:  FEFFFFFFFF9F1258 R9:  000000007FFE3000 R10: 0000000000003EBD R11: 0000D9DE9457C8FA
 R12: 0000000000000000 R13: 0000000000000000 R14: 0000019583B72850 R15: 0000000000000000
 RIP: 00007FF6C6813215 EFLAGS: 00010202

 Bytes at instruction pointer:
 49 83 3C 38 00 49 8D 14 38 0F 84 8C 00 00 00 45 33 C0 48 8D 4A 08 48 39

Stack trace:
 FFFF0000 FF140000 FFF0D000 FF000000 FF000000 FF000000 C663DEDF 00007FF6
 00989680 00000000 C6813197 00007FF6 E73DCED0 00000195 E73DCE90 00000195
 00000000 00000000 00000000 00000000 00989680 00000000 C67AA674 00007FF6
 00000000 00000000 00000000 00000000 E73DCE90 00000195 00989680 00000000
 00000000 00000000 FFFFFFFF FFFFFFFF 00000000 00000000 10158443 00007FFB
 00000000 00000000 00000000 00000000 0000003D 00000000 00000040 00000000
 E5790394 00000195 E5790000 00000195 7FFE0358 00000000 1014BB47 00007FFB
 097ACD2C 00000000 1017C4DE 00007FFB 00090010 00000000 000003D0 00000000
 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000
 1DAFF7C0 00000082 0DC738A0 00007FFB C097A320 0000DA98 1DAFF770 00000082
 00000000 00000000 00000000 00000000 00000001 00000000 C691A9E2 00007FF6
 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000
 00000000 00000000 C691AA5E 00007FF6 E61987E2 01D72E57 00000000 00000000
 1DAFF878 00000082 00000000 00000000 1DAFF878 00000082 C691A689 00007FF6
 1DAFF818 00000082 00000000 00000000 1DAFF878 00000082 C691A87F 00007FF6
 00989680 00000000 E73DCED0 00000195 00000000 00000000 C6919FB7 00007FF6
 00000000 00000000 C65B5251 00007FF6 C097A32A 0000DA98 00989680 00000000
 00989680 00000000 C691A87F 00007FF6 7D5C58DA 0000D029 607224FC 00000000
 7D5C597A 0000D029 00989680 00000000 E73DCE90 00000195 C666A544 00007FF6
 C097A333 0000DA98 00000000 00000000 E73DCED0 00000195 E73DCE90 00000195
 E73DCE90 00000195 C666A58F 00007FF6 E73DCED0 00000195 00989680 00000000
 00000000 00000000 1DAFF890 00000082 ED360D28 00000195 C666AEE6 00007FF6
 ED360D20 00000195 01B2B7E4 00000000 19359AC5 00002983 1DAFF8A0 00000082
 00000000 00000000 C666B169 00007FF6 ED360D20 00000195 00000000 00000000

Operating system:
 Name:     Windows
 Release:  10.0.18362 ()
 Compiler: MSVC 1928

Configuration:
 Blitter:      40bpp-anim
 Graphics set: OpenGFX (7761)
 Language:     C:\Program Files (x86)\Steam\steamapps\common\OpenTTD\lang\russian.lng
 Music driver: dmusic
 Music set:    OpenMSX (7761)
 Network:      no
 Sound driver: xaudio2
 Sound set:    OpenSFX (7758)
 Video driver: win32-opengl

Fonts:
 Small:  Arial
 Medium: Arial
 Large:  Arial
 Mono:   sprite

AI Configuration (local: 0) (current: 0):
  0: Human
  1: Human
  2: Human
  3: Human
  4: Human
  5: Human

Libraries:
 LZMA:       5.2.5
 LZO:        2.10
 PNG:        1.6.37
 Zlib:       1.2.11

Module information:
 C:\Program Files (x86)\Steam\steamapps\common\OpenTTD\openttd.exe handle: 00007FF6C63E0000 size: 8542208 crc: EE6C3614 date: 2021-04-09 20:38:13
 C:\Windows\SYSTEM32\ntdll.dll handle: 00007FFB10140000 size: 1999968 crc: 4AC89A35 date: 2020-05-15 14:07:00
 C:\Windows\System32\KERNEL32.DLL handle: 00007FFB0FC70000 size: 722072 crc: 6FE00726 date: 2020-06-11 14:06:36
 C:\Windows\System32\KERNELBASE.dll handle: 00007FFB0DC10000 size: 2769000 crc: E036C143 date: 2020-05-15 14:07:01
 C:\Windows\System32\WS2_32.dll handle: 00007FFB0E730000 size: 442704 crc: 2DDECC74 date: 2019-11-12 23:24:27
 C:\Windows\System32\RPCRT4.dll handle: 00007FFB0FDD0000 size: 1170960 crc: 67AEB483 date: 2020-04-14 13:49:17
 C:\Windows\System32\IMM32.dll handle: 00007FFB0FF70000 size: 176152 crc: 2F06A7FC date: 2019-11-12 23:24:28
 C:\Windows\SYSTEM32\OPENGL32.dll handle: 00007FFADF2B0000 size: 1263616 crc: ECF4BAB8 date: 2019-11-12 23:26:12
 C:\Windows\System32\USER32.dll handle: 00007FFB0E3A0000 size: 1656904 crc: 8B366BCB date: 2020-05-15 14:06:00
 C:\Windows\System32\win32u.dll handle: 00007FFB0DA40000 size: 127064 crc: 99ED2E1A date: 2020-06-11 14:06:41
 C:\Windows\System32\msvcrt.dll handle: 00007FFB0E610000 size: 638072 crc: 162578A8 date: 2019-03-19 04:44:35
 C:\Windows\System32\ADVAPI32.dll handle: 00007FFB0EB00000 size: 654912 crc: A786DEEC date: 2020-04-17 16:41:07
 C:\Windows\System32\GDI32.dll handle: 00007FFB0FD30000 size: 147912 crc: D05A781B date: 2019-03-19 04:44:06
 C:\Windows\SYSTEM32\WINMM.dll handle: 00007FFB09CF0000 size: 129880 crc: 55B5B362 date: 2019-03-19 04:43:47
 C:\Windows\System32\sechost.dll handle: 00007FFB0E1F0000 size: 605896 crc: 49EA34BB date: 2020-04-14 13:49:17
 C:\Windows\System32\gdi32full.dll handle: 00007FFB0D0D0000 size: 1649152 crc: F9276D08 date: 2020-06-11 14:07:15
 C:\Windows\System32\msvcp_win.dll handle: 00007FFB0DA70000 size: 639400 crc: B49E61EB date: 2020-05-15 14:06:43
 C:\Windows\System32\combase.dll handle: 00007FFB0E7A0000 size: 3368104 crc: 7B75695E date: 2020-06-11 14:06:54
 C:\Windows\System32\ucrtbase.dll handle: 00007FFB0DB10000 size: 1023128 crc: 0CAF4E31 date: 2020-05-15 14:06:43
 C:\Windows\SYSTEM32\WINMMBASE.dll handle: 00007FFB09CC0000 size: 169632 crc: 8AE99BA1 date: 2019-03-19 04:43:47
 C:\Windows\System32\bcryptPrimitives.dll handle: 00007FFB0E170000 size: 524208 crc: 46E5F8B3 date: 2020-05-15 14:07:01
 C:\Windows\System32\cfgmgr32.dll handle: 00007FFB0D270000 size: 293344 crc: 692DD58F date: 2019-11-12 23:24:14
 C:\Windows\System32\SHELL32.dll handle: 00007FFB0EF00000 size: 7266080 crc: 580591D8 date: 2020-06-11 14:06:45
 C:\Windows\System32\shcore.dll handle: 00007FFB0EBD0000 size: 685368 crc: B22F34A1 date: 2020-05-15 14:06:01
 C:\Windows\SYSTEM32\GLU32.dll handle: 00007FFAFCBD0000 size: 163328 crc: F008179F date: 2019-11-12 23:26:13
 C:\Windows\System32\windows.storage.dll handle: 00007FFB0D2C0000 size: 7911176 crc: CD2AE25B date: 2020-06-11 14:06:40
 C:\Windows\System32\profapi.dll handle: 00007FFB0D050000 size: 133256 crc: 6530F1FB date: 2020-04-14 13:49:17
 C:\Windows\SYSTEM32\dxcore.dll handle: 00007FFB0BC70000 size: 112296 crc: FA96F575 date: 2019-03-19 04:44:06
 C:\Windows\System32\powrprof.dll handle: 00007FFB0D080000 size: 291336 crc: 5EA3DD21 date: 2019-03-19 04:44:35
 C:\Windows\System32\UMPDC.dll handle: 00007FFB0D000000 size: 54960 crc: DD4A4D6A date: 2019-03-19 04:43:49
 C:\Windows\System32\shlwapi.dll handle: 00007FFB0E340000 size: 329200 crc: AC7F2587 date: 2019-03-19 04:45:05
 C:\Windows\System32\kernel.appcore.dll handle: 00007FFB0D030000 size: 59088 crc: E549CD16 date: 2019-03-19 04:44:15
 C:\Windows\System32\cryptsp.dll handle: 00007FFB0DF20000 size: 80112 crc: 4C3EEAA9 date: 2019-03-19 04:44:33
 C:\Windows\System32\ole32.dll handle: 00007FFB0FFA0000 size: 1396152 crc: FB050B31 date: 2020-04-14 13:48:49
 C:\Windows\SYSTEM32\USP10.dll handle: 00007FFB09260000 size: 79360 crc: 936448BB date: 2019-12-02 15:24:59
 C:\Program Files (x86)\Steam\gameoverlayrenderer64.dll handle: 00007FFADDFE0000 size: 1808104 crc: 4E14C8C5 date: 2021-03-23 21:22:56
 C:\Windows\System32\PSAPI.DLL handle: 00007FFB0EBC0000 size: 18832 crc: BED9353E date: 2019-03-19 04:44:28
 C:\Windows\system32\uxtheme.dll handle: 00007FFB0B370000 size: 606720 crc: ABDDBC69 date: 2019-12-02 15:24:44
 C:\Windows\System32\MSCTF.dll handle: 00007FFB0FA60000 size: 1260744 crc: EEFC3CBA date: 2020-06-11 14:06:48
 C:\Windows\System32\OLEAUT32.dll handle: 00007FFB0FBA0000 size: 797464 crc: 6B4B4D84 date: 2020-06-11 14:06:55
 C:\Windows\SYSTEM32\atig6pxx.dll handle: 00007FFAFC2E0000 size: 92704 crc: 2A6F77A3 date: 2015-08-01 07:48:02
 C:\Windows\SYSTEM32\atio6axx.dll handle: 0000000059610000 size: 30769696 crc: A2665E0B date: 2015-08-01 07:48:06
 C:\Windows\System32\SETUPAPI.dll handle: 00007FFB0F5F0000 size: 4683784 crc: B6654CFE date: 2019-03-19 04:45:00
 C:\Windows\System32\bcrypt.dll handle: 00007FFB0E090000 size: 144376 crc: 49BE6FB0 date: 2019-09-09 17:44:42
 C:\Windows\SYSTEM32\VERSION.dll handle: 00007FFB04EB0000 size: 30680 crc: B07A186D date: 2019-03-19 04:45:05
 C:\Windows\SYSTEM32\dwmapi.dll handle: 00007FFB0B6D0000 size: 162384 crc: 1A143E3C date: 2019-09-09 17:44:41
 C:\Windows\SYSTEM32\atiadlxx.dll handle: 00007FFADDE90000 size: 1265184 crc: 7C401599 date: 2015-08-01 07:48:02
 C:\Windows\SYSTEM32\USERENV.dll handle: 00007FFB0CF20000 size: 140496 crc: C24AC543 date: 2019-11-12 23:24:14
 C:\Windows\SYSTEM32\PROPSYS.dll handle: 00007FFB099B0000 size: 979264 crc: 26938172 date: 2020-05-15 14:05:57
 C:\Windows\SYSTEM32\IPHLPAPI.DLL handle: 00007FFB0C580000 size: 229528 crc: AAC3D0D3 date: 2019-03-19 04:44:33
 C:\Windows\SYSTEM32\WTSAPI32.dll handle: 00007FFB09AD0000 size: 64792 crc: 945834EF date: 2019-03-19 04:44:33
 C:\Windows\SYSTEM32\DEVOBJ.dll handle: 00007FFB0CE00000 size: 159112 crc: E7C8A00D date: 2019-11-12 23:24:14
 C:\Windows\System32\WINTRUST.dll handle: 00000195E8B30000 size: 363624 crc: 33ACDA09 date: 2019-11-12 23:23:22
 C:\Windows\System32\MSASN1.dll handle: 00007FFB0D010000 size: 63040 crc: 011C8316 date: 2019-03-19 04:44:35
 C:\Windows\System32\CRYPT32.dll handle: 00007FFB0DF40000 size: 1330952 crc: 2B82D538 date: 2020-04-14 13:48:57
 C:\Windows\SYSTEM32\atig6txx.dll handle: 00007FFAFB2C0000 size: 174112 crc: 89B3CF32 date: 2015-08-01 07:48:02
 C:\Windows\SYSTEM32\TextInputFramework.dll handle: 00007FFB065C0000 size: 642216 crc: 8123E888 date: 2020-04-14 13:48:24
 C:\Windows\SYSTEM32\CoreUIComponents.dll handle: 00007FFB084E0000 size: 3327256 crc: E389630F date: 2019-09-09 17:44:29
 C:\Windows\SYSTEM32\CoreMessaging.dll handle: 00007FFB0B060000 size: 859944 crc: D8D55DA0 date: 2020-05-15 14:06:11
 C:\Windows\SYSTEM32\ntmarta.dll handle: 00007FFB0C060000 size: 181856 crc: 06EC83CA date: 2019-03-19 04:44:35
 C:\Windows\SYSTEM32\wintypes.dll handle: 00007FFB08F50000 size: 1393952 crc: 669DBF7A date: 2020-06-11 14:06:55
 C:\Windows\SYSTEM32\iertutil.dll handle: 00007FFB00D90000 size: 2774088 crc: 1002CA29 date: 2020-05-15 14:07:07
 C:\Windows\System32\clbcatq.dll handle: 00007FFB0E290000 size: 643752 crc: C1F090C7 date: 2019-03-19 04:44:30
 C:\Windows\System32\MMDevApi.dll handle: 00007FFB06930000 size: 449888 crc: A7B29868 date: 2019-11-12 23:22:32
 C:\Windows\SYSTEM32\AUDIOSES.DLL handle: 00007FFB069B0000 size: 1413712 crc: 5F9F289E date: 2020-05-15 14:03:28
 C:\Windows\SYSTEM32\dxgi.dll handle: 00007FFB0BD00000 size: 945192 crc: D0CB15EE date: 2020-05-15 14:05:28
 C:\Windows\SYSTEM32\d3d11.dll handle: 00007FFB0A2E0000 size: 2466304 crc: D7F29C88 date: 2019-11-12 23:23:06
 C:\Windows\SYSTEM32\aticfx64.dll handle: 00007FFB04EC0000 size: 1488816 crc: 8FB3014D date: 2015-08-01 07:48:44
 C:\Windows\SYSTEM32\atiuxp64.dll handle: 00007FFB04E80000 size: 172432 crc: 759DDFD5 date: 2015-08-01 07:48:44
 C:\Windows\SYSTEM32\atidxx64.dll handle: 00007FFB04320000 size: 12175400 crc: 06C2A0B9 date: 2015-08-01 07:48:46
 C:\Windows\SYSTEM32\apphelp.dll handle: 00007FFB0B240000 size: 564736 crc: DFFF1273 date: 2019-03-19 04:44:28
 C:\Windows\system32\mswsock.dll handle: 00007FFB0C870000 size: 408064 crc: 3EB5FC3F date: 2020-05-15 14:06:43
 C:\Windows\SYSTEM32\xaudio2_8.dll handle: 00000195E8BD0000 size: 3584 crc: AB91B441 date: 2019-03-19 04:43:47
 C:\Windows\SYSTEM32\xaudio2_9.DLL handle: 00007FFAED070000 size: 594944 crc: 571CE448 date: 2019-09-09 17:44:19
 C:\Windows\SYSTEM32\AVRT.dll handle: 00007FFB08E80000 size: 30672 crc: CC5AFD63 date: 2019-03-19 04:43:47
 C:\Windows\SYSTEM32\resourcepolicyclient.dll handle: 00007FFB0B6B0000 size: 70256 crc: 6783400B date: 2019-03-19 04:44:04
 C:\Windows\System32\dmusic.dll handle: 00007FFAFACD0000 size: 132608 crc: C9F5B874 date: 2019-03-19 04:45:38
 C:\Windows\System32\DSOUND.dll handle: 00007FFAE3670000 size: 602112 crc: C9C6543E date: 2019-03-19 04:43:47
 C:\Windows\System32\MSACM32.dll handle: 00007FFB06F50000 size: 107640 crc: C340459C date: 2019-03-19 04:43:47
 C:\Windows\System32\ksuser.dll handle: 00007FFB000A0000 size: 23264 crc: B862C2FB date: 2019-03-19 04:43:47
 C:\Windows\SYSTEM32\wdmaud.drv handle: 00007FFAFA230000 size: 253952 crc: 0D3BE49A date: 2019-03-19 04:43:47
 C:\Windows\SYSTEM32\msacm32.drv handle: 00007FFB0AAB0000 size: 29184 crc: DCE0859B date: 2019-03-19 04:43:47
 C:\Windows\SYSTEM32\midimap.dll handle: 00007FFB0AAA0000 size: 25600 crc: 1A533078 date: 2019-03-19 04:43:47
 C:\Windows\System32\dmsynth.dll handle: 00007FFAF9FD0000 size: 122880 crc: 408A0AD2 date: 2019-03-19 04:45:38
 C:\Windows\System32\Windows.UI.dll handle: 00007FFAFC160000 size: 1385176 crc: BBBF6156 date: 2020-05-15 14:04:35
 C:\Windows\System32\InputHost.dll handle: 00007FFB06220000 size: 1150784 crc: 772C49FD date: 2020-05-15 14:05:44

---- gamelog start ----
Tick 0: new game started
Revision text changed to 1.10.3, savegame version 218, not modified, _openttd_newgrf_version = 0x1a086d64
New game mode: 1 landscape: 2
Tick 1280: game loaded
New game mode: 2 landscape: 2
Tick 1280: game loaded
New game mode: 1 landscape: 2
Tick 39224: game loaded
New game mode: 2 landscape: 2
Tick 39224: game loaded
New game mode: 1 landscape: 2
Tick 60493: settings changed
Setting changed: vehicle.never_expire_vehicles : 0 -> 1
Tick 52147: game loaded
New game mode: 2 landscape: 2
Tick 52147: game loaded
New game mode: 1 landscape: 2
Tick 39493: game loaded
New game mode: 2 landscape: 2
Tick 39493: game loaded
New game mode: 1 landscape: 2
Tick 29222: game loaded
New game mode: 2 landscape: 2
Tick 29222: game loaded
New game mode: 1 landscape: 2
Tick 29222: game loaded
Revision text changed to 1.11.0, savegame version 290, not modified, _openttd_newgrf_version = 0x1b086d64
New game mode: 0 landscape: 2
---- gamelog end ----

Recent news messages:

*** End of OpenTTD Crash Report ***

Decoded stack trace:
[00] openttd              0x00007FF6C6813215
[01] openttd              0x00007FF6C6813197
[02] openttd              0x00007FF6C67AA674
[03] openttd              0x00007FF6C666A544
[04] openttd              0x00007FF6C666A58F
[05] openttd              0x00007FF6C666AEE6
[06] openttd              0x00007FF6C666B169
[07] openttd              0x00007FF6C69796D0
[08] KERNEL32             0x00007FFB0FC87BD4 BaseThreadInitThunk + 20
[09] ntdll                0x00007FFB101ACE51 RtlUserThreadStart + 33

*** End of additional info ***
```

**Comment by James103** [comment]
Can you please attach the corresponding crash.zip (with the crash.log, crash.png, crash.dmp, and (if available) crash.sav)?

**Comment by Booeraque** [comment]
yes, sure. where can I find it?

**Comment by James103** [comment]
On Windows 10, you can usually find the crash files in `%USERPROFILE%/Documents/OpenTTD`.

**Comment by Booeraque** [comment]
is that it?

**Comment by LordAro** [comment]
If you attached it to the email, it hasn't come through (GH doesn't accept dmp files, it needs to be in a zip)

**Comment by James103** [comment]
@Booeraque Are you still able to reproduce this crash? If so, can you please attach the crash files here by doing the following?
1. Create a ZIP and add in your `crash.log`, `crash.dmp`, and `crash.png`.
2. Attach that ZIP file in a comment here.

**Comment by LordAro** [comment]
Likely duplicate of #8967. Should be fixed in 1.11.2, released now!
